src/utils/plugins.ts

```diff
@@ -0,0 +1,24 @@
+import { FastifyRequest } from 'fastify'
+
+export function generateForwardHeaders(
+  forwardHeaders: string | undefined,
+  request: FastifyRequest,
```

Comment:
instead of passing the entire request object you can simply pass `request.headers` making the function easier to test and more versatile